File: LXDSlideMenu-master/LXDSlideMenuObjectiveC/LXDSlideMenuObjectiveC/LXDCommonViewController.m
//
//  LXDCommonViewController.m
//  LXDSlideMenuObjectiveC
//
//  Created by linxinda on 15/11/29.
//  Copyright © 2015年 sindriLin. All rights reserved.
//

#import "LXDCommonViewController.h"
#import "LXDSlideManager.h"

@interface LXDCommonViewController ()

@end

@implementation LXDCommonViewController

- (void)viewDidLoad {
    [super viewDidLoad];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}

- (IBAction)clickView:(id)sender {
    [[LXDSlideManager sharedManager] openOrClose];
}

@end

File: LXDSlideMenu-master/LXDSlideMenuSwift/LXDSlideMenuSwift/LXDCommonViewController.swift
//
//  LXDCommonViewController.swift
//  LXDSlideMenuSwift
//
//  Created by linxinda on 15/11/29.
//  Copyright © 2015年 sindriLin. All rights reserved.
//

import UIKit

class LXDCommonViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    @IBAction func click(sender: AnyObject) {
        LXDSlideManager.sharedManager().openOrClose()
    }

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */

}
